### PR TITLE
Match admin notes

### PR DESCRIPTION
**Pull request by hburn7**
Depends on:
- #408

Adds admin notes for matches

### CODE REVIEW

**Review by alixdotsh — APPROVED** [review]
meowdy, code looks good to me 🐱 👍

File: Database/Entities/Match.cs [hunk 1]
```diff
@@ -117,8 +117,17 @@ public class Match : UpdateableEntityBase, IAuditableEntity<MatchAudit>, IProces
     /// </summary>
     public ICollection<RatingAdjustment> PlayerRatingAdjustments { get; set; } = new List<RatingAdjustment>();
 
+    /// <summary>
+    /// A collection of <see cref="MatchAudit"/>s, one for each change made to this entity
+    /// </summary>
```
Comment (myssto) [inline Database/Entities/Match.cs:122]: Technically this isn't required, it inherits the generic documentation from the interface

File: Database/Entities/Match.cs [hunk 2]
```diff
@@ -117,8 +117,17 @@ public class Match : UpdateableEntityBase, IAuditableEntity<MatchAudit>, IProces
...
     public ICollection<MatchAudit> Audits { get; set; } = new List<MatchAudit>();
 
+    /// <summary>
+    /// A collection of <see cref="AdminNotes"/>s. Admins may leave notes on matches detailing
+    /// important information
+    /// </summary>
```
Comment (myssto) [inline Database/Entities/Match.cs:128]: See above comment

**Review by myssto — APPROVED** [review]
Approving but see comments

**Pull request merged**